versions.json

```diff
@@ -13,6 +13,12 @@
       "currentMajor": true,
       "previousMinor": true
     },
+    {
+      "version": "8.13.5",
+      "branch": "8.13",
+      "currentMajor": true,
+      "previousMinor": true
```

Comment:
shouldn't this be false?

Reply:
We want it set to true until 8.14 goes out for backport reasons 😃